crates/loro-wasm/src/lib.rs

```diff
@@ -4834,6 +4836,17 @@ impl UndoManager {
         Ok(executed)
     }
 
+    // Start a new grouping of undo operations.
```

Comment:
You should add docs here. Or `skip_typescript` here and add docs to the typescript declaration directly